pkgs/servers/jellyseerr/default.nix

```diff
@@ -0,0 +1,86 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, mkYarnPackage
+, nodejs
+, nodePackages
+, sqlite
+, fetchYarnDeps
+, python3
+}:
+
+let
+  pin = lib.importJSON ./pin.json;
+in
```

Comment:
Why not inline this? There are many packages that do this in nixpkgs where commands can be copied.

Reply:
Hi! I don't really understand, what do you recommend?

Reply:
I guess it is for the commodity of the ```update.sh``` script!